src/DataContainer/FacebookNode.php

```diff
@@ -29,8 +29,8 @@
 
 class FacebookNode implements FrameworkAwareInterface, ContainerAwareInterface
 {
-    use FrameworkAwareTrait;
     use ContainerAwareTrait;
+    use FrameworkAwareTrait;
```

Comment:
This change seems unrelated.

Reply:
see: https://github.com/m-vo/contao-facebook-import/pull/33